charts/seed-controlplane/charts/etcd/templates/etcd.yaml

```diff
@@ -0,0 +1,74 @@
+apiVersion: druid.gardener.cloud/v1alpha1
+kind: Etcd
+metadata:
+  name: etcd-{{ .Values.role }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    gardener.cloud/role: controlplane
+    role: {{ .Values.role }}
+{{- if .Values.labels }}
+{{ toYaml .Values.labels | indent 4 }}
+{{- end }}
+spec:
+{{- if .Values.podAnnotations }}
+  annotations:
+{{ toYaml .Values.podAnnotations | indent 4 }}
+{{- end }}
+  selector:
+    matchLabels:
+      garden.sapcloud.io/role: controlplane
```

Comment:
Can we use `gardener.cloud/role` everywhere?

Reply:
Done